packages/exoflex/src/components/DateTimePicker/DateTimePicker.web.tsx

```diff
@@ -129,10 +129,20 @@ export function DatePicker(props: PickerProps) {
         renderArrow={arrow}
       />
       <View style={styles.touchableActionWrapper}>
-        <TouchableRipple onPress={onCancel} style={styles.touchableAction}>
+        <TouchableRipple
+          accessibilityHint="Close the picker"
```

Comment:
We're not going to expose the props? In case devs need to use a different language.

Reply:
Hmm. Now you mention it, I didn't expose both `CANCEL` and `CONFIRM` text to the props either.

Maybe I should expose them and just remove `accessibilityHint`, what do you think?

Reply:
Sure, in case the devs don't implement a11y at all there are no surprises for them.

Reply:
Even if we don't give the button an `accessibilityLabel`, the system is actually smart enough to use the button text as the label.

Reply:
That's okay if we expose the texts. Currently, we're locked on to English.